### ISSUE TITLE
fswin was updated and it breaks the code

### ISSUE THREAD

**Issue by fpronto** [stevenvachon/hidefile#10]
I think your code is broken because of fswin update

I was trying to use the isHidden function but it couldn't identify my hidden files correctly, I think its because you are trying to find a "hidden" and it doesn't exists, fswin gives a IS_HIDDEN at this moment

I think I can do a pull request later this week, but I its easier for you to change it

Great work thanks

**Comment by stevenvachon** [comment]
Yeah, that project [doesn't correctly use semver](https://github.com/xxoo/node-fswin/issues/43). I've released a `winattr@3.0.1` (which this package depends on) that fixes it for Node 8.

I've also released `winattr@4.0.0` and _**`hidefile@4.0.0`**_.
